griptape/tasks/actions_subtask.py

```diff
@@ -103,7 +101,9 @@ def run(self) -> BaseArtifact:
             else:
                 results = self.execute_actions(self.actions)
 
-                self.output = ListArtifact([TextArtifact(format_output(r)) for r in results])
+                self.output = ListArtifact(
+                    [TextArtifact(name=f"{r[0]} output: {r[1].to_text()}", value=r[1].to_text()) for r in results]
```

Comment:
Is this the correct approach? Using `name` doesn't quite feel right.

Reply:
I think `name` should just be `{r[0]} output`.